CoffeeChat/Controller/TalkingPointsViewController.swift

```diff
@@ -104,14 +89,18 @@ class TalkingPointsViewController: UIViewController {
         skipButton.backgroundColor = .none
         skipButton.addTarget(self, action: #selector(skipButtonPressed), for: .touchUpInside)
         view.addSubview(skipButton)
+        
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(dismissKeyboard))
+        tapGesture.cancelsTouchesInView = false
+        view.addGestureRecognizer(tapGesture)
 
         setupConstraints()
     }
     
-    override func viewWillAppear(_ animated: Bool) {
+    override func viewDidAppear(_ animated: Bool) {
         getAllTalkingPoints()
         getUserTalkingPoints()
-        super.viewWillAppear(animated)
+        super.viewDidAppear(animated)
```

Comment:
Same here with putting this at the top. Again, idk what's better, but whatever you decide just be consistent.